app/src/main/java/com/infomaniak/swisstransfer/ui/components/transfer/TransferItemList.kt

```diff
@@ -23,7 +23,6 @@ import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
```

Comment:
Commenting only to block the PR while waiting for @LunarX review about my modifications.

Reply:
I approve the changes